content/docs/v3.1/benchmarks/etcd-2-2-0-benchmarks.md

```diff
@@ -69,3 +69,5 @@ The performance is calulated through results of 100 benchmark rounds.
 - Write QPS to cluster leaders seems to be increased by a small margin. This is because the main loop and entry apply loops were decoupled in the etcd raft logic, eliminating several blocks between them.
 
 - Write QPS to all members seems to be increased by a significant margin, because followers now receive the latest commit index sooner, and commit proposals more quickly.
+
+[hack-benchmark]: https://github.com/etcd-io/etcd/tree/master/hack/benchmark
```

Comment:
Change `master` to the latest `v3.1.x` tag, here and elsewhere under `/docs/v3.1/`.

Reply:
should this be v3.2, this looks like a v2.2 benchmark.

Reply:
I'd like to address this in https://github.com/etcd-io/website/issues/147